pages/activity-log.tsx

```diff
@@ -20,6 +22,18 @@ interface Props {
 }
 
 const ActivityLog: NextPage<Props> = ({ actions }) => {
+  useEffect(() => {
+    if (router.query.flash === "success") {
+      window.history.replaceState(null, "", "/embedded");
+      toast.success(router.query.message as string, {
+        id: router.query.message as string,
+        duration: 4000,
+      });
+    } else if (router.query.flash === "error") {
+      window.history.replaceState(null, "", "/embedded");
+      toast.error(router.query.message as string, { id: "error" });
+    }
+  }, []);
```

Comment:
Do we want to go to embedded here or stay on activity log?

Reply:
Need to double check code when copying and pasting